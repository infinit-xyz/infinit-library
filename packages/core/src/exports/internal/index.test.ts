import { expect, test } from 'vitest'

import * as internal from './index'

test('exports - internal/index: should match snapshot', () => {
  expect(Object.keys(internal)).toMatchSnapshot()
})
