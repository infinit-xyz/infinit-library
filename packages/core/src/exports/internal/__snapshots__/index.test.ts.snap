// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`exports - internal/index: should match snapshot 1`] = `
[
  "validateActionData",
  "zodAddress",
  "zodAddressNonZero",
  "zodHex",
  "ContractVerifier",
  "verifyContract",
  "BlockExplorerParams",
  "ContractInfo",
  "ContractVerifierCallback",
]
`;
